source/module_ri/module_exx_symmetry/irreducible_sector.cpp

```diff
@@ -0,0 +1,238 @@
+#include "module_ri/module_exx_symmetry/irreducible_sector.h"
+
+namespace ModuleSymmetry
+{
```

Comment:
I would suggest adding some comments to these codes in this source file

Reply:
There have been many comments in the corresponding header telling what each function does.